check_source.py

```diff
@@ -171,9 +172,14 @@ def check_source_submission(self, source_project, source_package, source_revisio
             )
 
             try:
-                add_role_msg = 'Created automatically from request %s' % self.request.reqid
-                add_role_reqid = create_add_role_request(self.apiurl, source_project, self.required_maintainer,
-                                                         'maintainer', message=add_role_msg)
+                add_roles = get_request_list(self.apiurl, source_project, req_state=['new'], req_type='add_role')
+                add_roles = list(filter(self.select_add_role, add_roles))
```

Comment:
Doesn't that mean any random open addrole request will be used? There is s chance that somebody else is already requesting managership, without it being 'our' request

Reply:
That's the job of the filter, so I don't think so.

But the request may be in `review` if the devel project has reviewers set.

Reply:
good point, I was not sure about review state. So I think it makes sense to add it.
@DimStar77 maybe better filter name can be used. I just do not have a good name in my mind that explain it selects add role that has maintainer for user or group set to required user.

Reply:
I made a suggestion below about the name